### ISSUE TITLE
Different training styles between the original PhaseNet (Zhu etal. 2018) and seisbench PhaseNet

### ISSUE THREAD

**Issue by JUNZHU-SEIS** [seisbench/seisbench#126]
I notice that the original PhaseNet is trained using TensorFlow with a loss function called tf.nn.softmax_cross_entropy_with_logits(logits, target_probability)
However, seisbench PhaseNet use loss(preds, target_probability), instead.
Although they are almost the same in terms of mathematics, I wonder whether there exists any difference between them. A similar question related to this problem is https://discuss.pytorch.org/t/nn-model-best-practices-should-it-output-logits-or-probabilities/12848

**Comment by yetinam** [comment]
Thanks for pointing this out. Indeed, the only difference is numerical stability with the logit opion being more favourable from that perspective.

Closing the issue in favour of #127. Feel free to reopen if necessary.
